src/js/components/Home/Home.js

```diff
@@ -20,13 +18,22 @@ class Home extends Component {
 
     return (
       <div className="Home">
-      <Navbar titleFromParent={loc_navBarTitle} navbarItems={loc_navbarItems}/>
-      <h4> Home Component </h4>
+        <Navbar
+          titleFromParent={loc_navBarTitle}
+          navbarItems={loc_navbarItems}
+        />
+        <br/>
 
-      <NavLink to="/Books"> <button className="btn btn-success"> Books </button> </NavLink>
-      <h4> Insert speedy picture here </h4>
-      <Mapbuttons />
-      {/*<ArticleApp/>*/}
+        <NavLink to="/Books">
+          {" "}
+          <button className="btn btn-success"> Books </button>{" "}
+        </NavLink>
+        <br/>
+        <br/>
+        <br/>
```

Comment:
using line breaks to add padding... nay

Reply:
ya... this is horrible

Reply:
I have fixed this , thank you for your feed back.